addons/mail/static/src/core/common/channel_member_model.js

```diff
@@ -121,6 +121,14 @@ export class ChannelMember extends Record {
               })
             : undefined;
     }
+
+    get totalUnreadMessages() {
+        let counter = this.message_unread_counter;
```

Comment:
Nitpick: getter looks like a relational many fields rather than count. Suffix count(er) would help clarify it